job-server/src/main/scala/spark/jobserver/io/JobCassandraDAO.scala

```diff
@@ -223,7 +223,7 @@ class JobCassandraDAO(config: Config) extends JobDAO with FileCacher {
     val tuples = JListWrapper(rows).toIndexedSeq.map { row =>
       (row.getInt(ChunkIndex), row.getBytes(Binary).array())
     }
-    tuples.sortBy(_._1).toMap.values.foldLeft(Array[Byte]()) { _ ++ _ }
```

Comment:
.toMap will make it unsorted again

We actually don't need to sort, because ChunkIndex is clustered column